02_bubble/sarahdavy1983/main.go

```diff
@@ -0,0 +1,40 @@
+package main
+
+import (
+	"fmt"
+)
+
+func bubbleSort(s []int) []int {
+	sliceCopy := append(s[:0:0], s...)
```

Comment:
Should use `var SliceCopy = s` or ` Slicecopy := s` to copy slice

Reply:
also there isn't any reason why you needed sliceCopy to start with; you can do it in place

Reply:
@Joshcarp 
Thanks for the feedback, I have updated as per most of your comments.  I just want to clarify what you mean by " also there isn't any reason why you needed sliceCopy to start with; you can do it in place"?  Per my understanding of the requirements we need to create a copy